site/source/tutorials/reactive_programming.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: false
+title: Reactive programming
+overview: Dojo 2 embraces reactive programming patterns like top-down property propagation. This article gives a high level overview of the reactive patterns in Dojo 2 and how they are different from more traditional patterns.
+---
+# Reactive programming
+
+Reactive programming is concernedwith handling change - specifically, the _propagation of change_. As web applications have become more and more asynchronous, keeping track of application state has become increasingly complex. With traditional programming patterns, we rely on event listeners or try to keep track of application state manually.
```

Comment:
not sure about "more and more asynchronous", can we be more specific about the challenges (i don't think it's just asynchronous flow)